Northstar.CustomServers/mod.json

```diff
@@ -53,6 +53,11 @@
 		{
 			"Name": "ns_allow_kill_commands",
 			"DefaultValue": "0"
+		},
+		{
+			"Name": "ns_progression_enabled",
+			"DefaultValue": "0",
+			"Flags": "ARCHIVE_PLAYERPROFILE"
```

Comment:
thinking about it, this could've just been a modsettings toggle..
I get if you don't want to change it now though

Reply:
Personally I'm indifferent about where this goes. I don't really want the user to be able to enable and disable it from outside the lobby normally